torchvision/models/detection/faster_rcnn.py

```diff
@@ -361,4 +362,5 @@ def fasterrcnn_resnet50_fpn(pretrained=False, progress=True,
         state_dict = load_state_dict_from_url(model_urls['fasterrcnn_resnet50_fpn_coco'],
                                               progress=progress)
         model.load_state_dict(state_dict)
+        overwrite_eps(model, 0.0)
```

Comment:
I would have added a comment here explaining why we are doing this, so that users reading the code understand that this is to keep BC

Reply:
done